### DISCUSSION TITLE
Looking to change the git clone http path

### DISCUSSION THREAD

**Discussion by AlecMBurton** [gogs/gogs#6895] [Q&A]
I'm looking to change the git clone path for a new install. I currently have everything set up and the path for my repositories is currently:

https://gogs.domain.com/ORG/repo.git

We are migrating from gitweb and gitolite and there are a number of scripts and other code that uses the format:

https://gogs.domain.com/repo.git

Is there any way that I can accomplish the necessary changes with gogs? Thanks!

**Comment by unknwon** [comment]
Hey, it is currently not possible to archive this directly with Gogs, and this is just something incompatible with the basic design of the Gogs. I saw someone posted a workaround with some Apache configuration, but not sure how well that would work out.

**Reply by AlecMBurton** [reply]
Do you know where I can find the possible workaround?

**Reply by unknwon** [reply]
Ohh.. forgot to mention it was on the Discord. Copy pasting here:

>Yes, you can, with rewrite rules, don't remember the specifics
https://stackoverflow.com/questions/1828987/apache-mod-rewrite-for-shorter-urls So the thing here is that you will have to go thru the apache documentation's instructions to understand the basics before creating your own rules.
